cli/src/commands/next.rs

```diff
@@ -56,9 +56,15 @@ pub(crate) struct NextArgs {
     offset: u64,
     /// Instead of creating a new working-copy commit on top of the target
     /// commit (like `jj new`), edit the target commit directly (like `jj
-    /// edit`).
-    #[arg(long, short)]
+    /// edit`). Takes precedence over config in `ui.movement.edit`; i.e.
+    /// will negate `ui.movement.edit = "never"`
```

Comment:
nit: split paragraphs and omit period from the summary line. `jj next -h` only shows the summary line.
```
    /// Instead of creating a new working-copy commit on top of the target
    /// commit (like `jj new`), edit the target commit directly (like `jj
    /// edit`)
    ///
    /// Takes precedence over config in `ui.movement.edit`; i.e.  will negate
    /// `ui.movement.edit = "never"`.
```

fix the other help messages as well.

Reply:
Ah, these two files were inconsistent with other commands. I've spot checked a few other commands like `new`, `init`, etc, and they don't have periods. I've removed all the periods at the end of sentences in `next` and `prev` as well. Thanks for catching.